### PR TITLE
Ghostery: 3.2.0

### PR DESCRIPTION
**Pull request by smalluban**
Fixes https://github.com/ghostery/journal/issues/147

Use rebase when merging.

Additional changes:
* Builds on Apple silicon
* uses ASDF to manage nodejs and ruby version
* Basic CI with Github Action
* hardcodes signing certificates for Ghostery release 
* update swiftlint

### CODE REVIEW

**Review by philipp-classen — APPROVED** [review]

File: UserAgent.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -5615,7 +5615,7 @@
 			);
 			runOnlyForDeploymentPostprocessing = 0;
 			shellPath = /bin/sh;
-			shellScript = "export NODE_BINARY=`asdf which node`\n./node_modules/react-native/scripts/react-native-xcode.sh\n";
+			shellScript = "export NODE_BINARY=`readlink -f ~/.asdf/shims/node`\n\n./node_modules/react-native/scripts/react-native-xcode.sh\n";
```
Comment (philipp-classen) [inline UserAgent.xcodeproj/project.pbxproj:5618]: No problem for me, but note that "readlink -f" is a GNU extension. Out of the box, not available on MacOS.

I have not tested it, but "realpath" might be the safer way on MacOS. (On Linux, it is also there.)

Comment (chrmod) [reply UserAgent.xcodeproj/project.pbxproj]: thanks for suggestion!

**Comment by GRadziejewski** [comment]
# Iteration nr 1.

## Smoke tests on desktop
- 

## Smoke tests on mobile
- [ ] Browser: Ghostery Private Browser 3.2.0 (3)
	OS: iOS 16.6 (iPhone 13)

  ### Checked features
  * [x] Install fresh (no profiles and etc)
  * [ ] Update 
  * [x] Ghostery cannot work before onboarding
  * [x] Onboarding
~~Enable/Disable Ghostery~~
  * [x] Enable/Disable Ad-blocking
  * [x] Enable/Disable Anti-tracking
  * [x] Pause Ghostery
  * [x] Never Consent
  * [ ] Trackers Preview 
  * [ ] Select/Unselect any/all tracker/s
  * [ ] Sign in/out
      * [ ] Sign in and then update
      ~~Check Sync from / into another device~~
		~~Check Ghostery Settings logs~~
		~~Check Ghostery Background logs~~
  * [ ] Check account settings
  * [ ] Log into Google account and open google.com
      * [ ] Try to search anything
      * [ ] Try to search something with video
  * [ ] Log into FB
  * [ ] Update of the GBE
  * [ ] Other (from PR)


## Checked PRs / bugfixes
- 

## New issues
-

**Comment by GRadziejewski** [comment]
Tests for iteration nr 1 are finished.

**Comment by GRadziejewski** [comment]
Tests for iteration nr 1 are finished.

**Pull request merged**